orttraining/orttraining/python/training/ortmodule/_io.py

```diff
@@ -9,6 +9,18 @@
 import torch
 import warnings
 
+
+class _OutputIdentityOp(torch.autograd.Function):
```

Comment:
let's add a comment to explain by this is needed. 
and we should deprecate this after moving to torch 1.9